source/clear-linux/tutorials/kata.rst

```diff
@@ -16,6 +16,8 @@ This tutorial assumes you have installed |CL| on your host system.
 For detailed instructions on installing |CL| on a bare metal system, visit
 the :ref:`bare metal installation tutorial<bare-metal-install>`.
 
+If you have Clear Containers installed on your |CL| system, then follow the :ref:`Migrate Clear Containers to Kata Containers\* tutorial<kata_migration>`.
+
```

Comment:
Just some nits:
* Can you split the line after "follow the"? 
* The asterisk after Kata Containers isn't needed, since it has already been referenced earlier in the doc.
* The "M" in "Migrate Clear ..." should be lowercase.

Reply:
@mvincerx: sure, thanks for taking care of this.

Reply:
@intelkevinputnam
Could you please open a new PR and apply these changes, including adding the note proposed by @joselamego ?

Reply:
On it.

Reply:
Thanks much. :)